pkg/activator/net/revision_backends.go

```diff
@@ -273,20 +273,14 @@ func (rw *revisionWatcher) checkDests(dests sets.String) {
 
 func (rw *revisionWatcher) run(probeFrequency time.Duration) {
 	var dests sets.String
-	var timer *time.Timer
+	timer := time.NewTicker(probeFrequency)
```

Comment:
I think this badboy definitely needs to be stopped in `rw.doneCh` case to not leak forever. Not sure if its automatically collected otherwise tbh.